bower.json

```diff
@@ -9,7 +9,7 @@
     "type": "git",
     "url": "https://github.com/purescript-contrib/purescript-avar.git"
   },
-  "main": "",
+  "main": [],
```

Comment:
@JordanMartinez 
`language-purescript` fails to parse this because its a string instead of an array.
I'm assuming its an issue with this package and not the parser, (because the parser works on every other package I've tried so far).

https://github.com/JordanMartinez/purescript-language-purescript/blob/v0.1.1/src/Web/Bower/PackageMeta.purs#L68

Reply:
:thinking: We may still want to fix this upstream in `lang-purescript` because when we reimport the Registry, they may be using the new spago, and it'll fail here. But perhaps @f-f or @thomashoneyman could clarify whether that's what would happen in the Registry reimport.

Reply:
Ah yes indeed, that will happen. So let's fix it upstream